UPGRADE.md

```diff
@@ -0,0 +1,145 @@
+# UPGRADING 
+## [From 7.0.0-alpha1 to Unreleased] 
+### [shopsys/project-base] 
+#### Upgrade instructions     

```

Comment:
Personally I would remove this subtitle because all these instructions are "Upgrade instructions"